diff --git a/src/auth/index.ts b/src/auth/index.ts
index ab2a6b882988746e342e58ca54edc5e0c37a4585..2325cbf6277145be5d07e85c0e9f8faf8e0add9a 100644
--- a/src/auth/index.ts
+++ b/src/auth/index.ts
@@ -94,8 +94,7 @@ const refreshExpiredIdToken = async (
     headers: {
       "Content-Type": "application/x-www-form-urlencoded",
     },
-    body: `grant_type=refresh_token&refresh_token=${refreshToken}`,
-    cache: "no-store",
+    body: `grant_type=refresh_token&refresh_token=${refreshToken}`
   });
 
   if (!response.ok) {